src/util/system.cpp

```diff
@@ -447,14 +448,24 @@ const fs::path& ArgsManager::GetDataDir(bool net_specific) const
         path /= fs::PathFromString(BaseParams().DataDir());
 
     if (fs::create_directories(path)) {
-        // This is the first run, create wallets subdirectory too
-        fs::create_directories(path / "wallets");
+        // This is the first run, create wallets subdirectory and bitcoin.conf
```

Comment:
this comment should probably go in the docs for `SetupDatadir`

Reply:
Not sure about this. This comment was already present. I just added `and bitcoin.conf` in it.

Reply:
But you moved the code the comment was "attached to" into that new function...

Reply:
Moved in https://github.com/bitcoin/bitcoin/pull/23931/commits/ccbd9a68c00d1588a8846aa43a6f2b3777255393